src/SPC/builder/unix/library/imap.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\unix\library;
+
+use SPC\exception\FileSystemException;
+use SPC\exception\RuntimeException;
+use SPC\exception\WrongUsageException;
+
+trait imap
+{
+    /**
+     * @throws FileSystemException
+     * @throws RuntimeException
+     * @throws WrongUsageException
+     */
+    protected function build(): void
+    {
+        // ext-imap is NOT thread safe, do not build with ZTS builds
+        if ($this->builder->getOption('enable-zts')) {
```

Comment:
we don't have a "lib-conflicts" option so this is what I came up with

Reply:
I'm not sure if this is actually true. The php docs say it's not thread safe, but there are references in the extension source. I've uncommented it for now.